zingocli/tests/utils/mod.rs

```diff
@@ -158,8 +158,14 @@ pub mod scenarios {
                         }),
                 );
             }
-            pub fn build_and_launch(funded: Option<String>, custom_conf: Option<String>) -> Self {
-                let mut sb = if let Some(conf) = custom_conf {
+            pub fn build_and_launch(
+                funded: Option<String>,
+                zingo_wallet_dir: Option<String>,
+            ) -> Self {
+                if let Err(e) = LightClient::init_logging() {
+                    eprintln!("Can't initiate logging: {e}")
```

Comment:
Even with the `Once` I added to `init_logging`, this will a: Cause a panicking test to potentially poison the once and cause other tests to fail, and b: Dump all logging from all tests to the terminal concurrently, in a way that's impossible to tell what's from what test.

I think the correct solution here is [tracing_test](https://docs.rs/tracing-test/latest/tracing_test/).